src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitInventory;
+import core.basesyntax.service.ReportGenerator;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String REPORT_HEADER = "fruit,quantity" + System.lineSeparator();
+
+    @Override
+    public void generate(String path) {
+        File file = new File(path);
+        try {
```

Comment:
Creating a new file is unnecessary if you just want to write or append to it. FileWriter or BufferedWriter will create it if it doesn't exist.